.github/pull_request_template.md

```diff
@@ -0,0 +1,27 @@
+<!-- Replace {issue_number} with the issue that will be closed after merging this PR -->
+## Description
+Fixes #{issue_number}: A few sentences describing the changes proposed in this pull request.
+
+## Type of change
+Please delete options accordingly to the description.
+
+<!-- Write an `x` in all the boxes that apply -->
+- [ ] Bug fix (non-breaking change which fixes an issue)
+- [ ] New feature (non-breaking change which adds functionality)
+- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
+- [ ] This change requires a documentation update
+
+
+### Suggested Checklist
+<!-- You do not need to complete all the items by the time you submit the pull request, but most likely the changes will only be merged if all the tasks are done. -->
+
+<!-- Write an `x` in all the boxes that apply -->
+- [ ] I have performed a self-review of my own code.
+- [ ] I have made corresponding changes to the documentation.
+- [ ] My changes generate no new warnings.
+- [ ] I have commented my code, particularly in hard-to-understand areas.
+- [ ] I have read the [CONTRIBUTING](https://github.com/UCLH-Foundry/PIXL/blob/main/CONTRIBUTING.md) docs
+- [ ] If you are developing in your local host, please check that your code passes the following the style guidelines `pre-commit run -a` (this is also tested in CI).
+- [ ] If you are developping in your local host, please check that your code is properly tested with `pytest` (this is also tested in CI).
+- [ ] I have marked this PR as draft. Otherwise, you are good to go for PR review by selecting `UCLH-Foundtry/arc-dev`
+- [ ] Make sure your branch is up-to-date with main branch. See [CONTRIBUTING](https://github.com/UCLH-Foundry/PIXL/blob/main/CONTRIBUTING.md) for a general example if `rebase` is need.
```

Comment:
we squash merge so probably doesn't need to have the rebase mention

Reply:
resolved here 43d3337e3bb644d554f94296cc23abe7fd250615